### PR TITLE
more tests and remove buggy prime_power_form

### PR DESCRIPTION
**Pull request by simonbrandhorst**

### CODE REVIEW

File: src/QuadForm/Spaces.jl [hunk 1]
```diff
@@ -29,22 +29,22 @@ function hom(V::AbsSpace, W::AbsSpace, B::MatElem; check::Bool = false)
...
+    fl = B * GW * transpose(_map(B, involution(W))) == GV
     if !fl
       error("Matrix does not define a morphism of spaces")
     end
   end
   return AbsSpaceMor(V, W, B)
 end
 
-function image(f::AbsSpaceMor, v::Vector)
+function image(f::AbsSpaceMor, v::Union{Vector,Matrix})
```
Comment (simonbrandhorst) [inline src/QuadForm/Spaces.jl:40]:
```suggestion
function image(f::AbsSpaceMor, v::Vector)
```

File: src/QuadForm/Spaces.jl [hunk 2]
```diff
@@ -29,22 +29,22 @@ function hom(V::AbsSpace, W::AbsSpace, B::MatElem; check::Bool = false)
...
   V = domain(f)
   w = matrix(base_ring(V), 1, length(v), v) * f.matrix
   return collect(w)
```
Comment (simonbrandhorst) [inline src/QuadForm/Spaces.jl:43]:
```suggestion
  return vec(collect(w))
```

File: src/QuadForm/QuadBin.jl [hunk 3]
```diff
@@ -442,24 +393,25 @@ function isequivalent(f::QuadBin{fmpz}, g::QuadBin{fmpz}; proper::Bool = true)
   if isindefinite(f)
     fred = reduction(f)
     gred = reduction(g)
-    if issquare(d)
```
Comment (thofma) [inline src/QuadForm/QuadBin.jl:445]: I guess this part makes no sense? :-D

Comment (simonbrandhorst) [reply src/QuadForm/QuadBin.jl:445]: yes `_isequivalent_reducible` has taken care of this.

Comment (simonbrandhorst) [reply src/QuadForm/QuadBin.jl:445]: so that part of  code is never executed

Comment (thofma) [reply src/QuadForm/QuadBin.jl:445]: So, should we remove it?

Comment (simonbrandhorst) [reply src/QuadForm/QuadBin.jl:445]: o.k. after some further tests I am convinced that the `_isequivalent_reducible` works.

**Comment by simonbrandhorst** [comment]
waits for equality check in #541 to be merged

**Pull request merged**